packages/openapi3/test/tsp-openapi3/parameters.test.ts

```diff
@@ -0,0 +1,155 @@
+import { assert, describe, expect, it } from "vitest";
+import { tspForOpenAPI3 } from "./utils/tsp-for-openapi3.js";
+
+describe("converts top-level parameters", () => {
```

Comment:
Also, are there any valid parameter names in OpenAPI3 that would not be valid model names in TypeSpec? Thinking specifically of reserved names, like 'model', which would need to be escaped.

Reply:
Any reserved names (like `model`) are already escaped. There's a spec that verifies that but it doesn't include typespec reserved words, I can add a test for that.

Reply:
Added tests for invalid names as well.